app/Http/Controllers/Api/V1/SurveyController.php

```diff
@@ -75,7 +75,8 @@ public function update(Survey $survey, UpdateSurveyRequest $request): JsonRespon
             'Survey' => new SingleSurveyResource(
```

Comment:
```suggestion
            'survey' => new SingleSurveyResource(
```
Waarom gebruik je hier de naam: SingleSurveyResource en niet gewoon SurveyResource?